extension/src/extension.ts

```diff
@@ -1,4 +1,4 @@
-import { commands, env, ExtensionContext, ViewColumn } from 'vscode'
```

Comment:
As mentioned in https://github.com/iterative/vscode-dvc/pull/3994#issuecomment-1569567208, we shouldn't be using env in the scenario.